spec/fixtures/marc_to_solr/991583506421.mrx

```diff
@@ -42,145 +41,120 @@
   </datafield>
-  <datafield tag="555" ind1=" " ind2=" ">
+  <datafield ind1=" " ind2=" " tag="555">
     <subfield code="a">No. 1 (1974)-8 (1975), with no. 8.</subfield>
   </datafield>
-  <datafield tag="650" ind1=" " ind2="0">
+  <datafield ind1=" " ind2="0" tag="650">
     <subfield code="a">Printing</subfield>
     <subfield code="z">United States</subfield>
     <subfield code="x">History</subfield>
     <subfield code="v">Periodicals.</subfield>
   </datafield>
-  <datafield tag="650" ind1=" " ind2="0">
+  <datafield ind1=" " ind2="0" tag="650">
     <subfield code="a">Printing</subfield>
     <subfield code="v">Periodicals.</subfield>
     <subfield code="0">http://id.loc.gov/authorities/subjects/sh85106764</subfield>
   </datafield>
-  <datafield tag="610" ind1="2" ind2="0">
+  <datafield ind1="2" ind2="0" tag="610">
     <subfield code="a">American Printing History Association</subfield>
     <subfield code="v">Periodicals.</subfield>
   </datafield>
-  <datafield tag="655" ind1=" " ind2="7">
+  <datafield ind1=" " ind2="7" tag="655">
     <subfield code="a">Periodicals.</subfield>
     <subfield code="2">lcgft</subfield>
     <subfield code="0">http://id.loc.gov/authorities/genreForms/gf2014026139</subfield>
   </datafield>
-  <datafield tag="710" ind1="2" ind2=" ">
+  <datafield ind1="2" ind2=" " tag="710">
     <subfield code="a">American Printing History Association.</subfield>
     <subfield code="0">http://id.loc.gov/authorities/names/n79074225</subfield>
   </datafield>
-  <datafield tag="785" ind1="0" ind2="0">
+  <datafield ind1="0" ind2="0" tag="785">
     <subfield code="t">APHA newsletter</subfield>
     <subfield code="w">(DLC)sn 86021679</subfield>
     <subfield code="w">(OCoLC)13154156</subfield>
   </datafield>
-  <datafield tag="999" ind1=" " ind2=" ">
+  <datafield ind1=" " ind2=" " tag="999">
     <subfield code="a">158</subfield>
   </datafield>
-  <datafield tag="950" ind1=" " ind2=" ">
-    <subfield code="b">2020-12-02 20:58:51 US/Eastern</subfield>
-    <subfield code="a">false</subfield>
-  </datafield>
-  <datafield tag="852" ind1="8" ind2="1">
-    <subfield code="b">rare</subfield>
-    <subfield code="c">ga</subfield>
-    <subfield code="h">2006-0030Q</subfield>
-    <subfield code="k">Oversize</subfield>
-    <subfield code="8">22262098640006421</subfield>
+  <datafield ind1=" " ind2=" " tag="INT">
+    <subfield code="a">P</subfield>
   </datafield>
-  <datafield tag="866" ind1=" " ind2="0">
-    <subfield code="a">No. 1 (Feb. 1975)-no. 68</subfield>
-    <subfield code="8">22262098640006421</subfield>
+  <datafield ind1=" " ind2=" " tag="INST">
+    <subfield code="a">01PRI_INST</subfield>
   </datafield>
-  <datafield tag="866" ind1=" " ind2="0">
-    <subfield code="z">LACKS: no. 25-26,29,49-50, 56</subfield>
+  <datafield ind1=" " ind2=" " tag="AVA">
+    <subfield code="0">991583506421</subfield>
     <subfield code="8">22262098640006421</subfield>
```

Comment:
I'm assuming you created the mrx using the bibdata bibliographic endpoint. Please use the sets in alma to export the desired fixtures.